### PR TITLE
Add Kestra plugin

### PR DESCRIPTION
**Pull request by anna-geller**

### CODE REVIEW

**Review by davidgasquez — COMMENTED** [review]
Left a suggestions based on the official [Awesome list guidelines](https://github.com/sindresorhus/awesome). Basically removing the links from the description and making the first letter uppercase.

Does it sounds good to you?

File: README.md [hunk 1]
```diff
@@ -44,6 +44,8 @@
 - [data load tool - DuckDB destination](https://dlthub.com/docs/dlt-ecosystem/destinations/duckdb) - Extract and load data from APIs to DuckDB using dlt.
 - [target-duckdb](https://hub.meltano.com/loaders/target-duckdb/) - Load data to DuckDB based on Singer spec.
 - [Airbyte DuckDB destination](https://docs.airbyte.com/integrations/destinations/duckdb/) - Load data to DuckDB with Airbyte.
+- [Kestra DuckDB plugin](https://kestra.io/plugins/plugin-jdbc-duckdb) — run queries with DuckDB to [schedule data transformations](https://demo.kestra.io/ui/blueprints/community?selectedTag=35) and process automations, and [run event-driven anomaly detection pipelines](https://motherduck.com/blog/motherduck-kestra-etl-pipelines/)
```
Comment (davidgasquez) [inline README.md:47]:
```suggestion
- [Kestra DuckDB plugin](https://kestra.io/plugins/plugin-jdbc-duckdb) — Run queries with DuckDB to schedule data transformations and process automations, and run event-driven anomaly detection pipelines.
```

**Comment by anna-geller** [comment]
thanks so much, this was so fast!

**Comment by davidgasquez** [comment]
Thank you for the PR!

**Pull request merged**